src/generators/java/presets/CommonPreset.ts

```diff
@@ -97,6 +98,74 @@ private String toIndentedString(Object o) {
 }`;
 } 
 
+function renderMarshalProperties(renderer: JavaRenderer, model:CommonModel) {
+  const properties = model.properties || {};
+  const propertyKeys = [...Object.keys(properties)];
+  const marshalProperties = propertyKeys.map(prop => {
+    const formattedPropertyName = renderer.nameProperty(prop,model);
+    const modelInstanceVariable = `this.${formattedPropertyName}`;
+    return `if(${modelInstanceVariable} != null) {
+        propList.add("${formattedPropertyName}:"+${modelInstanceVariable}.toString());
+    }`;
+  });
+  return marshalProperties.join('\n');
+}
+/**
+ * Render `marshal` function based on model's properties
+ */
+function renderMarshalling({ renderer, model }: {
+  renderer: JavaRenderer,
+  model: CommonModel,
+}): string {
+  return `public String marshal() {
+  List<String> propList = new ArrayList();
+  ${renderer.indent(renderMarshalProperties(renderer,model))}
+  return propList.stream().collect(Collectors.joining(","));
+}`;
+}
+
+function renderUnmarshalProperties(renderer: JavaRenderer, model:CommonModel) {
+  const properties = model.properties || {};
+  const propertyKeys = [...Object.keys(properties)];
+  const unmarshalProperties = propertyKeys.map(prop => {
+    const formattedPropertyName = renderer.nameProperty(prop,model);
+    const setterFunction = `set${formattedPropertyName.charAt(0).toUpperCase()}${formattedPropertyName.slice(1)}`;
+    // eslint-disable-next-line
+    const getType = `jsonObject.get${properties[prop].type?.toString().charAt(0).toUpperCase()}${properties[prop].type?.toString().slice(1)}`;
```

Comment:
I am assuming these lines are the problems to eslint complaining?

```suggestion
    const getType = `jsonObject.get${FormatHelpers.upperFirst(properties[prop].type?.toString())}`
```

Was it the optional chaining of `properties[prop].type?`? 

In case it was, at the beginning of the map callback, you can add the following and then change all calls of ` properties[prop].type?` to `properties[prop].type`.

```ts
const propModel = properties[prop];
if(propModel.type === 'undefined') {
  Logger.error(`Could not render unmarshal for property ${prop}`);
  return; 
}
```